easybuild/easyconfigs/p/pytest-shard/pytest-shard-0.1.2-GCCcore-12.3.0.eb

```
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'pytest-shard'
+version = '0.1.2'
+
+homepage = 'https://github.com/AdamGleave/pytest-shard'
+description = """pytest plugin to support parallelism across multiple machines.
+
+Shards tests based on a hash of their test name enabling easy parallelism across machines,
+suitable for a wide variety of continuous integration services.
+Tests are split at the finest level of granularity, individual test cases,
+enabling parallelism even if all of your tests are in a single file
+(or even single parameterized test method).
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [('binutils', '2.40')]
+dependencies = [('Python', '3.11.3'),
+                ('Python-bundle-PyPI', '2023.06'),  # It checks for it after installed
+                ]
```

I think it needs `pytest`, which is included in the bundle? In that case, could you make the comment a bit more clear? Perhaps something like this (also fixed the style/indentation a bit):
```suggestion
dependencies = [
    ('Python', '3.11.3'),
    ('Python-bundle-PyPI', '2023.06'),  # provides pytest
]
```